x-pack/plugins/maps/public/shared/icons/additional_layer_icons.js

```diff
@@ -54,3 +54,9 @@ export const FillableRectangle = ({ style }) => (
     <rect width="15" height="15" x=".5" y=".5" style={style} rx="4"/>
   </svg>
 );
+
+export const ColorableLine = ({ style }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+    <rect width="16" height="4" y="6" style={style}/>
```

Comment:
Could we use an actual svg `line` or `path` with a `stroke-width` property? https://developer.mozilla.org/en-US/docs/Web/SVG/Element/line or https://www.w3schools.com/graphics/svg_stroking.asp